custom/icds_reports/reports/poshan_progress_dashboard_data.py

```diff
@@ -54,7 +54,7 @@ def calculate_aggregated_row(data, aggregation_level, data_period, unique_id):
     aggregated_row = calculate_percentage_single_row(deepcopy(aggregated_row))
     # rounding values
     for col in ['num_launched_districts', 'num_launched_blocks', 'num_launched_states']:
-        aggregated_row[col] = round(aggregated_row[col])
+        aggregated_row[col] = round(aggregated_row.get(col, 0))
     aggregated_row = prepare_structure_aggregated_row(deepcopy(aggregated_row),
```

Comment:
you sure, subsequent use of `aggregated_row` would not break with the same error?

Reply:
offlined about this